src/web/components/AuditTrail/AuditLog.tsx

```diff
@@ -5,12 +5,12 @@ import { getPrettyAuditDetails } from '../../services/auditTrailService';
 
 import './AuditTrail.scss';
 
-type AuditLogProps = Readonly<{
+type AuditTrailProps = Readonly<{
   log: AuditTrailDTO;
 }>;
 
-function AuditLog({ log }: AuditLogProps) {
-  const formattedEventData = getPrettyAuditDetails(log.event, log.eventData);
+function AuditLog({ log }: AuditTrailProps) {
```

Comment:
This function is still called AuditLog - is that intentional?

Reply:
yeah, kindof.  AuditTrail was already singular.  I renamed it AuditTrailRow to make it easier to find.